wandb/cli/cli.py

```diff
@@ -946,6 +947,16 @@ def _check_launch_imports():
     "as config to the compute resource. The exact content which should be "
     "provided is different for each execution backend. See documentation for layout of this file.",
 )
+@click.option(
```

Comment:
We should probably also make a note of this option in the cli command description